packages/webcomponents/src/components/business-forms/payment-provisioning/terms-and-conditions/business-terms-conditions-form-step.tsx

```diff
@@ -100,14 +99,13 @@ export class BusinessTermsConditionsFormStep {
     } else {
       onSuccess();
     }
-    this.submitted.emit({ data: response, metadata: { completedStep: BusinessFormStep.termsAndConditions } });
-    this.stepCompleted.emit({ data: response, formStep: BusinessFormStepV2.termsAndConditions });
+    this.stepCompleted.emit({ data: response, formStep: BusinessFormStep.termsAndConditions });
```

Comment:
Shouldn't this step emit the `submitted` event?

Reply:
Hey @TiagoFuelber - its actually the `payment-provisioning-core.tsx` file that emits the `submitted` event once a business provisioning request has been initiated. 

Thanks for fixing that typescript error!